enterprise/reporting/reporting.go

```diff
@@ -374,6 +384,23 @@ func (r *HandleT) mainLoop(ctx context.Context, clientName string) {
 	r.getMinReportedAtQueryTime = stats.Default.NewTaggedStat(StatReportingGetMinReportedAtQueryTime, stats.TimerType, tags)
 	r.getReportsQueryTime = stats.Default.NewTaggedStat(StatReportingGetReportsQueryTime, stats.TimerType, tags)
 	r.requestLatency = stats.Default.NewTaggedStat(StatReportingHttpReqLatency, stats.TimerType, tags)
+
+	lastReportedAt := time.Now().UTC().Unix() / 60
```

Comment:
Since this is being accessed from 2 go-routines, there can be a data race happening.
```suggestion
	lastReportedAt := atomic.NewInt64(time.Now().UTC().Unix() / 60)
```

get: lastReportedAt.Load()
set: lastReportedAt.Store(currentMs)